File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.java
   class Solution {
        public int countConsistentStrings(String allowed, String[] words) {
            int result = words.length;
            for (int i = 0; i < words.length; i++) {
                for (int k = 0; k < words[i].length(); k++) {
                    if (!allowed.contains(String.valueOf(words[i].charAt(k)))) {
                        result--;

                        break;
                    }
                }
            }
            return result;
        }
    }

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int mask = 0;
        for(auto ch: allowed){
            int bit = 1<<(ch-'a');
            mask = mask | bit;
        }
        int cnt = 0;
        for(auto word: words){
            bool flag = true;
            for(auto ch: word){
                int bit = 1<<(ch-'a');
                if((mask & bit) == 0){
                    flag = false;
                    break;
                }
            }
            if(flag) cnt++;
        }
        return cnt;
    }
};